File: foodies_api/transaksi/transaksi.php
<?php
include '../connection.php';

$id_summary = $_POST['id_summary'];
$trans_total = $_POST['trans_total'];
$trans_date = $_POST['trans_date'];
$trans_time = $_POST['trans_time'];
$jumlah_orang = $_POST['jumlah_orang'];
$id_rekening = $_POST['id_rekening'];
$kode_unik = $_POST['kode_unik'];
$user_id = $_POST['user_id'];


$sqlQuery = "INSERT INTO transaksi SET id_summary = '$id_summary', trans_total = '$trans_total', trans_date = '$trans_date',trans_time = '$trans_time', jumlah_orang = '$jumlah_orang', id_rekening = '$id_rekening', kode_unik = '$kode_unik', user_id = '$user_id'";

$resultOfQuery = $connectNow->query($sqlQuery);

if ($resultOfQuery) {
    echo json_encode(array("success" => true));
} else {
    echo json_encode(array("success" => false));
}

File: lib/users/model/transaksi.dart
class Transaksi {
  int? id_transaksi;
  int? id_summary;
  double? trans_total;
  String? trans_date;
  int? jumlah_orang;
  int? kode_unik;
  int? quantity;
  String? name_res;
  String? name;

  Transaksi({
    this.id_summary,
    this.id_transaksi,
    this.jumlah_orang,
    this.kode_unik,
    this.name_res,
    this.name,
    this.quantity,
    this.trans_date,
    this.trans_total,
  });

  factory Transaksi.fromJson(Map<String, dynamic> json) => Transaksi(
        id_transaksi: int.parse(json["id_transaksi"]),
        id_summary: int.parse(json["id_summary"]),
        jumlah_orang: int.parse(json["jumlah_orang"]),
        kode_unik: int.parse(json["kode_unik"]),
        quantity: int.parse(json["quantity"]),
        name: json["name"],
        name_res: json["name_res"],
        trans_total: double.parse(json["trans_total"]),
        trans_date: json['trans_date'],
      );
}
